packages/cli/assets/pluginPackage/src/dataProcess.ts

```diff
@@ -1,7 +1,6 @@
-import { DataProcessType, ArgsType, UniDataset } from '@pipcook/pipcook-core'
+import { DataProcessType, ArgsType, Sample } from '@pipcook/pipcook-core'
 
-const templateDataProcess: DataProcessType = async (data: UniDataset, args: ArgsType): Promise<UniDataset> => {
-  return {} as UniDataset;
+const templateDataProcess: DataProcessType = async (data: Sample, args: ArgsType): Promise<void> => {
```

Comment:
Does the local data-process type still have metadata for the second parameter? Cuz it still has the responsibility to change metadata as discussed